sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -264,7 +266,15 @@ vpn_subscriptions AS (
 )
 SELECT
   *,
-  CONCAT(plan_interval, "-", plan_currency, "-", (plan_amount / 100)) AS pricing_plan,
+  CONCAT(
+    plan_interval_count,
+    "-",
+    plan_interval,
+    "-",
+    plan_currency,
+    "-",
+    (plan_amount / 100)
+  ) AS pricing_plan,
```

Comment:
This is needed because the 6-month plan available in euros was displaying incorrectly as `month-eur-41.97` and `6-month-eur-41.94` is less misleading.

This change doesn't make `pricing_plan` isn't any more or less unique, so dashboard counts won't be impacted.

This does not require any backfilling (no downstream tables that only update date partitions use this field)